scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/extension/Cast.scala

```diff
@@ -92,21 +93,33 @@ private[extension] class CastMethodDefinitions(private val classesWithTyping: Ma
 
 }
 
-object CastMethodDefinitions {
-  private val stringClass = classOf[String]
+class CastImpl(target: Any, classLoader: ClassLoader, classesBySimpleName: Map[String, Class[_]]) extends Cast {
```

Comment:
It looks like we don't need the interface. WDYT about removing the Cast trait and renaming CastImpl to Cast?

Reply:
Left todo and I will check this later